observation_portal/proposals/serializers.py

```diff
@@ -46,6 +46,10 @@ def validate_emails(self, emails):
         return emails
 
 
+class ProposalInviteResponseSerializer(serializers.Serializer):
```

Comment:
A common pattern I adopted when we have separate request and response serializers is to denote them as such.